hash/apple2_flop_orig.xml

```diff
@@ -6328,4 +6328,216 @@
 
+	<software name="tipnflip">
+		<description>Tip 'N Flip</description>
+		<year>1984</year>
+		<publisher>Sunburst Communications</publisher>
+		<info name="release" value="2019-07-29"/>
+		<sharedfeat name="compatibility" value="A2P,A2E,A2EE,A2C,A2GS" />
+		<!-- It requires a 48K Apple ][+ or later. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="234782">
+				<rom name="tip 'n flip.woz" size="234782" crc="d51ac3ae" sha1="b5e542b1d3855f16168979d529e703e6fabacf8f" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="qsrtnhcl">
+		<description>Return of Heracles</description>
+		<year>1983</year>
+		<publisher>Quality Software</publisher>
+		<info name="release" value="2019-08-02"/>
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS" />
+		<!-- It runs on any Apple II with 48K. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="235508">
+				<rom name="return of heracles.woz" size="235508" crc="50be7d16" sha1="24e16ab619a269e8cb47d0a3557aec1cf9c35d98" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="racemdnt">
+		<description>Race For Midnight</description>
+		<year>1981</year>
+		<publisher>Avant-Garde Creations</publisher>
+		<info name="release" value="2019-08-02"/>
+		<sharedfeat name="compatibility" value="A2P,A2E,A2EE,A2C,A2GS" />
+		<!-- It requires a 48K Apple ][+ or later. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="235492">
+				<rom name="race for midnight.woz" size="235492" crc="3eae9c48" sha1="9d5c8f15c28880c9888b47d7b4682708a954c742" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="blkmagic">
+		<description>Black Magic</description>
+		<year>1987</year>
+		<publisher>Datasoft</publisher>
+		<info name="release" value="2019-08-02"/>
+		<sharedfeat name="compatibility" value="A2P,A2E,A2EE,A2C,A2GS" />
+		<!-- It requires a 64K Apple ][+ or later. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="252868">
+				<rom name="black magic.woz" size="252868" crc="9579ff50" sha1="50106741bfb8c7a78adb0b1c40b39d166509fcda" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mablmans">
+		<description>Mabel's Mansion</description>
+		<year>1983</year>
+		<publisher>Datamost</publisher>
+		<info name="release" value="2019-08-02"/>
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS" />
+		<!-- It runs on any Apple II with 48K. -->
+		<!-- (Note: many emulators have difficulty booting this game properly.
+		Also, if you have a Z80 card or a Mockingboard in any slot,
+		try removing it.) -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="235468">
+				<rom name="mabel's mansion.woz" size="235468" crc="53e139cb" sha1="aa9d6189808697aa4381f899399ebddb5a1d988f" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="alabyrnt">
+		<description>Labyrinth (Activision)</description>
```

Comment:
The full title of this one is "Labyrinth: The Computer Game." That it was developed by Lucasfilm Games should also be mentioned.

Reply:
Adjusted the title accordingly.